src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -2222,13 +2222,14 @@ instruct varray_equalsC(iRegP_R11 ary1, iRegP_R12 ary2, iRegI_R10 result,
 %}
 
 instruct vstring_compareU(iRegP_R11 str1, iRegI_R12 cnt1, iRegP_R13 str2, iRegI_R14 cnt2,
-                          iRegI_R10 result, vReg_V1 v1, vReg_V2 v2, vReg_V3 v3, vReg_V4 v4, vReg_V5 v5,
+                          iRegI_R10 result, vReg_V1 v1, vReg_V2 v2, vReg_V3 v3, vReg_V4 v4,
+                          vReg_V5 v5, vReg_V6 v6, vReg_V7 v7,
                           vRegMask_V0 v0, iRegP_R28 tmp1, iRegL_R29 tmp2)
```

Comment:
Hi,
When StrCompNode is `StrIntrinsicNode::UU` or `StrIntrinsicNode::LL`,  if-branch (element_compare) will be executed in `C2_MacroAssembler::string_compare_v`, and `lmul` is set to 2, so v6 and v7 are not used in `string_compareL` and `string_compareU`, here is the code:
C2_MacroAssembler::string_compare_v
```
  if (str1_isL == str2_isL) { // LL or UU
    element_compare(str1, str2, zr, cnt2, tmp1, tmp2, v2, v4, v1, encLL, DIFFERENCE);
    j(DONE);
  } else { // LU or UL
    Register strL = encLU ? str1 : str2;
    Register strU = encLU ? str2 : str1;
```
C2_MacroAssembler::element_compare
```
 bind(loop);
  vsetvli(tmp1, cnt, sew, Assembler::m2);
  vlex_v(vr1, a1, sew);
  vlex_v(vr2, a2, sew);
```

Reply:
Right, thank you